man/systemd-vmspawn.xml

```diff
@@ -97,26 +97,20 @@
     </refsect2>
 
     <refsect2>
-      <title>User Namespacing Options</title>
+      <title>System Identity Options</title>
```

Comment:
seems you swapped order of a section you added in the previous commit? should probably be move there, not here